src/main/java/org/gridsuite/sensitivityanalysis/server/dto/ContingencyListExportResult.java

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
why not use the usual license header with two ** ?
`/**
 * Copyright (c) 2024, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */
`
same for author:
`/**
 * @author author...
 */
`

Reply:
done